.changes/unreleased/Fixes-20230320-153225.yaml

```
@@ -0,0 +1,6 @@
+kind: Fixes
+body: add negative part_number arg for split part macro
+time: 2023-03-20T15:32:25.5932-05:00
+custom:
+  Author: dave-connors-3
+  Issue: "615"
```

I just created #7915 so updating the changelog entry accordingly:
```suggestion
  Issue: "7915"
```